types/index.d.ts

```diff
@@ -14,6 +14,8 @@ export interface MaterialTableProps<RowData extends object> {
     onRowAdd?: (newData: RowData) => Promise<any>;
     onRowUpdate?: (newData: RowData, oldData?: RowData) => Promise<any>;
     onRowDelete?: (oldData: RowData) => Promise<any>;
+    onRowAddCancelled?: () => void;
```

Comment:
Actually, onRowAddCancelled & onRowUpdateCancelled has rowData Parameter

Reply:
Thanks for approving and merging the code.
Shall I add the rowData parameter while calling onRowAddCancelled and onRowUpdateCancelled functions?

Reply:
I try to use onRowAddCancelled but it does nothing.
Does it still work?